etsi_its_conversion/etsi_its_conversion/src/Converter.cpp

```diff
@@ -127,8 +83,10 @@ const std::string Converter::kBtpDestinationPortOffsetParam{"btp_destination_por
 const int Converter::kBtpDestinationPortOffsetParamDefault{8};
 const std::string Converter::kEtsiMessagePayloadOffsetParam{"etsi_message_payload_offset"};
 const int Converter::kEtsiMessagePayloadOffsetParamDefault{78};
-const std::string Converter::kEtsiTypesParam{"etsi_types"};
-const std::vector<std::string> Converter::kEtsiTypesParamDefault{"cam", "cpm_ts", "denm"};
+const std::string Converter::kRos2UdpEtsiTypesParam{"ros2udp_etsi_types"};
+const std::string Converter::kUdp2RosEtsiTypesParam{"udp2ros_etsi_types"};
+const std::vector<std::string> Converter::kRos2UdpEtsiTypesParamDefault{"cam", "cam_ts", "cpm_ts", "denm"};
+const std::vector<std::string> Converter::kUdp2RosEtsiTypesParamDefault{"cam", "cpm_ts", "denm"};
```

Comment:
Again why aren't we also adding `cam_ts` to the default parameter set, which basically includes all technical possibilites. If we don't want to add them in `Udp2Ros` then I also would not add them in `Ros2Udp` and strictly stick to EN's by default if available

Reply:
keep and mention in README as discussed